src/Exceptions/OAuthServerException.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Laravel\Passport\Exceptions;
+
+use Exception;
+use Illuminate\Http\Response;
+use League\OAuth2\Server\Exception\OAuthServerException as LeagueException;
+
+class OAuthServerException extends Exception
+{
+    protected $response;
+
+    public function __construct(LeagueException $e, Response $response)
```

Comment:
Add a DocBlock here like on other constructors.

Reply:
We use fully qualified namespaces in docblocks everywhere and two spaces before and after a type for `@param` annotations.